src/interpreter/symbolTable.ts

```diff
@@ -113,6 +116,11 @@ export const addFunctionSymbolTableEntries = (
     getNameFromDeclaratorPattern(node.id),
     symbolTable
   );
+  if (!isFunctionSymbolTableEntry(functionEntry)) {
```

Comment:
I thought you only want user defined functions for this, and not builtins?

Reply:
This is the symbol table entry for user-defined functions. Should I rename `FunctionSymbolTableEntry` to `UserDeclaredFunctionSymbolTableEntry`?

Reply:
oh okay anything